hello-world-python/README.md

```diff
@@ -0,0 +1,110 @@
+# Conclave Hello World Python Sample
+
+This sample shows how to execute a python code inside a conclave enclave.
+Please note this sample uses the hello world sample which is available as part of the conclave sdk.
+Below ``Configurations to be added to a hello world conclave sample`` section talks about the configurations to be added 
+to this sample to enable it to execute python code.
+Conclave support for Python is provided by the polyglot capabilities in GraalVM. 
+Refer to the GraalVM [documentation](https://www.graalvm.org/reference-manual/embed-languages/) on embedding languages 
+for detailed instructions on how to use this capability.
+
+When you use the graalvm-native-image runtime you have the option to enable Python language support. 
+When enabled you can create a context in which you can load and execute a Python source module.
+
+# What are the different modes in which you can run a typical Conclave application
+
+There are four modes in which an enclave can be loaded
+1. Release Mode - You need a real Intel SGX CPU. You load the enclave in release mode when you are in production.
+2. Debug Mode - You need a real Intel SGX CPU. This provides you with certain capabilities whcih enables you to debug
+   your enclave application.
+3. Simulation Mode - This does not require you to have an Intel SGX CPU. This can be used to test your conclave 
+   applications when in development mode. This mode requires a linux env. This can be run inside a Docker container 
+   on Mac. Enclave code is compiled to native binary and loaded inside a separate JVM inside your host JVM.
+4. Mock Mode - Enclave code can be directly run inside the host VM. This is very fast, and hence should ideally be used
+   for day to day rapid development and testing.
+
+To read more about modes go to the [docs](https://docs.conclave.net/enclave-modes.html).
+
+# Configurations to be added to a hello world conclave sample
+
+If you have not yet built any conclave application, take a look at this 
+[page](https://docs.conclave.net/writing-hello-world.html) which talks about how to run a simple hello world
+conclave application.
+
+To build the application using conclave init tool take a look [here](https://docs.conclave.net/conclave-init.html)
```

Comment:
The tool "Conclave Init" is not used to build applications. We use Gradle. I am assuming what you want to say is something in the lines "To build the application, you can follow the steps  present in [here](https://docs.conclave.net/conclave-init.html#run-the-application). Notice the hash tag. That will direct people to the correct section on the page.

Reply:
removing reference to conclave init as of now